src/.schema.sql

```diff
@@ -0,0 +1,15 @@
+CREATE SCHEMA kubercade AUTHORIZATION nrhwqomw
```

Comment:
This is the user you created correct? I saw elsewhere that we needed to set AUTHORIZATION so I did so

Reply:
That's the user - and as far as I can tell the authorization looks right